docs/02-Configure-a-Project/01-Project-Configuration-Files.md

```diff
@@ -251,7 +251,7 @@ Fields:
 -   `name`: an identification string for the variant
 -   `display_name`: how the variant is displayed in the Evergreen UI
 -   `run_on`: a list of acceptable distros to run tasks for that variant
-    on. The first distro in the list is the primary distro. The others
+    a. The first distro in the list is the primary distro. The others
```

Comment:
nit: I think you accidentally changed "on" to "a"

Reply:
Fixed.